src/CatalogGraphQLClient.php

```diff
@@ -69,7 +108,21 @@ public function call(string $functionName, array $arguments, array $selectionSet
      *
      * @throws NonExistingFieldException
      */
-    public function find(array $searchBody, array $fields, int $page = 1, $perPage = 100): PaginatedProducts
+    public function find(array $searchBody, array $fields, int $page = 1, int $perPage = self::DEFAULT_PER_PAGE): PaginatedProducts
+    {
+        return new PaginatedProducts($this->findItems(Types\Query::GET_PRODUCTS, $searchBody, $fields, $page, $perPage));
```

Comment:
If you're going to deprecate this method then just make it a proxy to the other
```suggestion
        return $this->findProducts($searchBody, $fields, $page, $perPage);
```